examples/riverpod_navigator2/linux/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Could you delete the linux folder? I typically don't commit them unless there's a specific change that needs to be kept, but that isn't the case here